src/components/carousel/index.js

```diff
@@ -229,11 +227,6 @@ export default class Carousel extends BaseComponent {
   };
 
   renderChildren() {
-    const {containerWidth} = this.state;
```

Comment:
By removing this, aren't u getting a flicker with the children width?

Reply:
No. Do you?

Reply:
I can check again, but the reason I added this code was to prevent this flicker. 
without it the children will render twice. The second time will correct the children width and its noticeable..